src/components/interface/dialog/DialogInput.jsx

```diff
@@ -25,6 +26,7 @@ const DialogInput = ({
     disabled={disabled}
     style={{ margin: '0 5px 25px 5px', ...style }}
     helperText={helperText}
+    InputLabelProps = {InputLabelProps}
```

Comment:
Why do we need these InputLabelProps?

Also remember that we don't need to follow the same syntax, we can convert ours to follow camelCase style.

This function could take in `inputLabelProps` and say `InputLabelProps={inputLabelProps}`

Reply:
Changed to camel case.

I added `InputLabelProps` to solve the overlap between the text field label and the value in the text field when the text field is not focused.